tests/fuzzer.c

```diff
@@ -1960,6 +1960,11 @@ static int basicUnitTests(U32 const seed, double compressibility)
         DISPLAYLEVEL(3, "OK \n");
     }
 
+    DISPLAYLEVEL(3, "test%3i : ZSTD_getSequences zeros : ", testNb++);
+    memset(CNBuffer, 0, 1000000);
+    assert(ZSTD_getSequences(ZSTD_createCCtx(), compressedBuffer, 1000000,
```

Comment:
this `assert()` produces a side effect and a memory leak.

Reply:
Perhaps a silly question:) What is the issue here?

Reply:
`ZSTD_createCCtx()` allocates a context, which is never freed afterwards, so it's a leak.
This is assuming that allocation is successful, otherwise it returns a `NULL` pointer, and using it triggers undefined behavior.

It's also a general rule that no `assert()` should ever trigger any side effect. Heap allocation is a side effect.